src/api/api.ts

```diff
@@ -0,0 +1,61 @@
+import { GetRequestParams, PostRequestParams } from './types';
+
+const backendName = 'https://ya-praktikum.tech/api/v2';
```

Comment:
```suggestion
const API_URL = 'https://ya-praktikum.tech/api/v2';
```
Я считаю, что константы лучше выделять такого вида названиями

Reply:
окей, поправила